app/api/v2/middleware/middleware.py

```diff
@@ -0,0 +1,31 @@
+from functools import wraps 
+from ..models.auth import UserModel 
+from flask import request
+from flask import jsonify, make_response, abort
+from flask_jwt_extended import (jwt_required, 
+                                verify_jwt_in_request, get_jwt_identity)
+
```

Comment:
Remove extra line

Reply:
@PaulKariukiRimiru  The extra line came about after linting my code according to pep8. Should I resolve it still?